media-gfx/gscan2pdf/gscan2pdf-2.5.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI 7, please.

Reply:
Alas perl-module.eclass is still `@SUPPORTED_EAPIS: 5 6`